ui/raidboss/data/03-hw/raid/a9s.js

```diff
@@ -131,30 +131,34 @@
       netRegexJa: NetRegexes.startsUsing({ source: 'リファビッシャー', id: '1A3C', capture: false }),
       netRegexCn: NetRegexes.startsUsing({ source: '废品翻新装置', id: '1A3C', capture: false }),
       netRegexKo: NetRegexes.startsUsing({ source: '재생자', id: '1A3C', capture: false }),
-      alertText: function(data) {
+      alertText: function(data, _, output) {
         if (data.mainTank == data.me)
           return;
-        return {
-          en: 'Get Behind',
-          de: 'Hinter ihn',
-          fr: 'Passez derrière',
-          ja: '背面へ',
-          ko: '보스 뒤로',
-          cn: '去背后',
-        };
+        return output.getBehind();
       },
-      infoText: function(data) {
+      infoText: function(data, _, output) {
         if (data.mainTank != data.me)
           return;
-        return {
+        return output.scraplineOnYou();
+        // ...probably, we hope...
```

Comment:
This is odd

Reply:
Removed.  It was intended to be a "you might not be the main tank anymore, but hopefully you still are! and if you're not, probably everybody else is in trouble!!"